.cspell.json

```diff
@@ -108,103 +108,122 @@
     "uvicorn"
   ],
   "ignoreWords": [
+    "prompty",
+    "AADSTS700082",
```

Comment:
why so many changes to this file?

Reply:
Reverted back to main's cspell, added the tests (which had the most words) to ignore in the cspell.json.